pkg/minikube/cluster/cluster.go

```diff
@@ -42,12 +42,12 @@ func init() {
 }
 
 // Bootstrapper returns a new bootstrapper for the cluster
-func Bootstrapper(api libmachine.API, bootstrapperName string) (bootstrapper.Bootstrapper, error) {
+func Bootstrapper(api libmachine.API, bootstrapperName string, nodeName string) (bootstrapper.Bootstrapper, error) {
```

Comment:
This feels like an abstraction violation: `cluster.Bootstrapper()` should not require a node be passed to it. It doesn't make sense which node one would select.

Reply:
You're absolutely right, I'm just using Bootstrapper to get at the command runner, which is per machine instead of per cluster. I think the solution would be to extract command runner out of bootstrapper, which would likely be a large enough change for its own PR.

Reply:
Hmm, Could you use `machine.Runner` instead?

Reply:
I suspect fixing this will involve moving command runner out of the bootstrapper entirely and changing how the bootstrapper interface works entirely.